lib/Text/Markup/rst2html_lenient.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
Why not `#!/usr/bin/env python3`?

Reply:
That might be a Debianism: We want to make sure that the binary from our system package is used (same for perl, btw). But this might not be universal, so I pushed a commit which changes the hashbang back.